nix-test/src/const.c

```diff
@@ -209,6 +209,31 @@ get_int_const(const char* err) {
     GET_CONST(ECANCELED);
 #endif
 
+#if defined(__OpenBSD__)
```

Comment:
You can revert the changes to this file.  This file has been obsolete ever since most constants moved into libc.  libc has its own, superior version.